teamA/jeongHaeMan1/routes/index.js

```diff
@@ -0,0 +1,9 @@
+//routes/index.js
+
+const express = require('express');
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router); // 같은 동일 선상의 routes 폴더 안에 존재하고 있는 userRouter.js 내용이 안에 잘 정의되어 있는 router라는 모듈을 긁어서 불러와서 발동시킨다
+
```

Comment:
개발 시 작성한 주석은 삭제해주세요 !
```suggestion
router.use("/users", userRouter.router);
```